R/search_users.R

```diff
@@ -49,15 +41,7 @@ search_users <- function(q, n = 100,
   
   stopifnot(is_n(n), is.atomic(q))
   if (n > 1000) {
-    warning(
-      "search only returns up to 1,000 users per ",
-      "unique search. Setting n to 1000..."
-    )
-    n <- 1000
-  }
-
-  if (nchar(q) > 500) {
-    stop("q cannot exceed 500 characters.", call. = FALSE)
+    abort("`n` must be <= 1,000")
```

Comment:
This error is not less informative than previously. Perhaps it was too verbose but maybe point to the documentation/reference?

Reply:
That's a different error — the old error about `q` will now be thrown because the twitter API will errror.

Reply:
Oh I see what you mean now that I'm on my computer. I'll make this a little more verbose.